impeller/typographer/font.h

```diff
@@ -9,11 +9,25 @@
 
 #include "fml/hash_combine.h"
 #include "impeller/base/comparable.h"
-#include "impeller/typographer/glyph.h"
+#include "impeller/geometry/scalar.h"
 #include "impeller/typographer/typeface.h"
 
 namespace impeller {
 
+//------------------------------------------------------------------------------
+/// @brief      Determines the axis along which there is subpixel positioning.
+///
+enum class AxisAlignment : uint8_t {
```

Comment:
This is find of course and not suggesting this be modified. But we also have type safe masks now. So this could be mask values with a mask defined as `using AxisAlignmentMask = Mask<AxisAlignment>;`.

Reply:
I'm not really using this as a Mask right now, at least I dont think. I think I could rewrite things a bit to treat axis alignment as an off or on mask for which of X/Y have subpixel positioning?

Reply:
Nah, this is fine.